validator.go

```diff
@@ -0,0 +1,45 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"log"
+	"os"
+	"path"
+
+	"github.com/gobuffalo/packr"
+	"github.com/xeipuuv/gojsonschema"
+)
+
+func main() {
+	var doc string
+	flag.StringVar(&doc, "doc", "examples/invalid/invalid_email.json", "Example file")
+	flag.Parse()
+	validate(doc)
+}
+
+func validate(doc string) {
+	dir, err := os.Getwd()
+	if err != nil {
+		log.Fatal(err)
+	}
+	doc = path.Join("file:///", dir, "/", "examples/invalid/invalid_email.json")
```

Comment:
`doc` shouldn't be overwritten here. It's set by the user and passed in the function, but then gets overwritten by the default.

Reply:
@stp ack